sdks/java/maven-archetypes/examples/src/main/resources/archetype-resources/pom.xml

```diff
@@ -101,6 +101,17 @@
         </plugin>
       </plugins>
     </pluginManagement>
+
+    <extensions>
+      <!-- Use os-maven-plugin to initialize the "os.detected" properties for Bigtable
```

Comment:
This is broken for runners which execute on a different OS than the client application that submitted the job.

Based upon reading http://netty.io/wiki/forked-tomcat-native.html, we should update sdks/java/io/google-cloud-platform/pom.xml to rely on the statically linked uber jar which will choose the correct library to load at runtime.